web/styles/modal.css

```diff
@@ -667,3 +667,67 @@
 #topic-summary-modal {
     width: 45em;
 }
+
+#onboarding-video-modal {
+    width: 97vw;
+    max-width: 1024px;
+
+    .modal__content {
+        overflow: hidden;
+    }
+
+    #onboarding-video-wrapper {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+
+        &::after {
+            content: "";
+            position: absolute;
+            background-image: url("../images/play_button.svg");
+            background-size: contain;
+            background-repeat: no-repeat;
+            background-position: center;
+            background-color: transparent;
+            width: 20%;
+            height: 20%;
+            pointer-events: none;
+        }
+
+        &.hide-play-button::after {
+            content: none;
+            background-image: none;
+        }
+    }
+
+    video {
```

Comment:
This should have a class on it; we don't want to ever write CSS for bare elements. (It has performance and readability/greppability issues).